### PR TITLE
Internalize some uikit classes to OC

### PR DESCRIPTION
**Pull request by pascalwengerter**
No changelog item since it (basically) only touches examples

### CODE REVIEW

File: src/components/OcSelect.vue [hunk 1]
```diff
@@ -97,7 +97,7 @@ For detailed documentation (props, slots, events, etc.), please visit https://vu
 
 ```js
 <template>
-  <div class="uk-width-medium">
+  <div style="width: 300px">
```
Comment (dschmidt) [inline src/components/OcSelect.vue:100]: Not sure what your policies are but some people consider inline style to be the devil's work :trollface:

Comment (dschmidt) [reply src/components/OcSelect.vue:100]: Is this a 1:1 replacement? or a change in behavior?

Comment (pascalwengerter) [reply src/components/OcSelect.vue]: Only happens inside the `<docs>` examples but yeah I've moved it to a dedicated class within the docs' SCSS files now. It's a 1:1 replacement of setting a fixed width for the select box

**Comment by dschmidt** [comment]
cool, lgtm - I'm just a bit reluctant to approve cause I'm not so familiar with this project/repo :)

**Comment by pascalwengerter** [comment]
> cool, lgtm - I'm just a bit reluctant to approve cause I'm not so familiar with this project/repo :) 

No worries, thought it could be reasonable to slowly involve you some more since you're starting to use the components if I'm not mistaken 😉 thanks for taking the time!

**Review by kulmann — APPROVED** [review]
Cool 👍

**Pull request merged**